manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Tutanota.Tutanota
+PackageVersion: 259.241213.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-12-16
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
This is probably an x64 application.